contracts/interchainTokenService/InterchainTokenService.sol

```diff
@@ -322,6 +323,8 @@ contract InterchainTokenService is IInterchainTokenService, AxelarExecutable, Up
         _deployRemoteStandardizedToken(tokenId, name, symbol, decimals, distributor, admin, destinationChain, gasValue);
     }
 
+    // TODO: pause check isn't needed for this since it's just passing tokens through, so we can save some gas
+    // TODO: Add commandID to help express vs execute reorder protection, i.e check if commandID is already executed
```

Comment:
Are we ok with the wait to generate a commandID for express? If we know a commandID can't we just execute?

Reply:
commandID is currently known when the contract call is made. It might still get reorged, but it's essentially hash of tx hash + event index + chain id. So it still helps protect in normal scenarios